### PR TITLE
Green Wallet Theme

### PR DESCRIPTION
**Pull request by cryptowd**
Changed color theme to green using the Ellaism logo to guide the color selection.
Feedback welcome.

Updated some references to Ethereum.
Removed references to Ledger and Digital Bitbox from front page.
Changed one menu item to refer to sending Ella rather than Ether.

### CODE REVIEW

**Comment by cryptowd** [comment]
This is an incremental update to the green theme.  This adds the logo to the index page using a .png file.

I hope I'm not making these changes a headache.  If I did anything incorrectly, let me know!

File: app/scripts/translations/en.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ NAV_InteractContract        : 'Interact with Contract ',
 NAV_Multisig                : 'Multisig ',
 NAV_MyWallets               : 'My Wallets ',
 NAV_Offline                 : 'Send Offline ',
-NAV_SendEther               : 'Send Ether & Tokens ',
+NAV_SendEther               : 'Send Ella & Tokens ',
```
Comment (ellaismer) [inline app/scripts/translations/en.js]: Right now can you revert this single line of change? We haven't decided the term yet.

Like Ethereum, we can call the money by the symbol: "You have 1.0 ELLA" or "You have 1.0 ETH", but for all currency terms like "Ether", "Finney", "Szabo", "Shannon" we share them with Ethereum.

File: app/styles/etherwallet-variables.less [hunk 2]
```diff
@@ -1,7 +1,18 @@
 // Colors
-@ether-navy                                  : #163151;
-@ether-navy-dark                             : #102138;
-@ether-blue                                  : #0e97c0;
+//@ether-navy                                  : #163151;
+//@ether-navy-dark                             : #102138;
+//@ether-blue                                  : #0e97c0;
```
Comment (ellaismer) [inline app/styles/etherwallet-variables.less]: Remove the comments if it is not used.

**Review by ellaismer — CHANGES_REQUESTED** [review]
Some minor changes requested. Everything else looks good!

**Review by cryptowd — COMMENTED** [review]
I've reverted the "Send Ella" in translations back to "Send Ether" and removed comments with old color values.
Glad everything else worked.

**Review by ellaismer — APPROVED** [review]

**Comment by ellaismer** [comment]
Thank you!

**Pull request merged**